src/internal/clock.ts

```diff
@@ -55,9 +54,10 @@ const performanceNowNanos = (function() {
   return () => origin + BigInt(Math.round(performance.now() * 1_000_000))
 })()
 const processOrPerformanceNow = (function() {
-  const processHrtime = hasProperty(process, "hrtime") && isFunction(process.hrtime.bigint) ?
-    process.hrtime :
-    undefined
+  const processHrtime =
```

Comment:
@tim-smart weird, they should be equivalent, what's the regression?

Reply:
```
> const predicate = (_) => true
undefined
> predicate(asdf)
Uncaught ReferenceError: asdf is not defined
> typeof asdf
'undefined'
```

You can use typeof on an undefined variable, but you can't use it for a function call.